index.html

```diff
@@ -3,47 +3,294 @@
+  {"Department":"Sucre","Region":4,"Victims":335825},
+  {"Department":"Tolima","Region":5,"Victims":190575},
+  {"Department":"Valle Del Cauca","Region":1,"Victims":528749},
+  {"Department":"Vaupes","Region":2,"Victims":6648},
+  {"Department":"Vichada","Region":3,"Victims":7380}];
+     // The ALL_CAPS treatment here is signalling that this variable is a constant - it should never change.
+  var MAX_VICTIMS = 1749491;
+  // This is where we'll store the data representing user interaction; event handlers (the functions that
+  // are called when a user clicks a button) work by updating the properties of this object. An alternative
+  // approach, where each event handler directly updates the table, would quickly become messy. Imagine,
+  // for example, that I've filtered the table to just show elementary schools. Next, I sort the table to
+  // be in descending order. Finally, I unfilter the table to show all schools. If I'm not careful, the
+  // newly revealed schools will not be sorted, because the filter operation doesn't know how to sort.
+  // Instead, updateTable() will sort, filter, and paginate based on whatever the options object contains,
+  // always starting from a fresh copy of the data, and we can count on the result to be consistent
+  // given the same values in options.
+  var options = {
+    sorted: false,
+    filtered: false,
+    page: 0
+  };
+  function updateTable() {
+    // Make a copy of the data
+    var data = conflict.slice();
+    // Sort conflict by victims
+    // 
+    // descending: points.sort(function(a, b){return b-a});
+
+    if (options.sorted) {
+      if (options.sorted === 'asc') {
+        
+        data.sort(function (a, b) { return a.Victims - b.Victims; });
+        // This anonymous function is responsible for comparing two values a and b. If it returns a positive
+        // number, we're saying that a should come after b in the array. A negative number means that b
+        // should come after a. A zero means that the two are equal. This function is using the 'enrollment'
+        // property of each observation of data for this comparison. You'll need to change 'enrollment' to
+        // whatever property you want to sort your data on.
+        //
+        // Also, note that we're sorting the data in place, changing, or mutating, the 'data' variable. See
+        // the note in the README about how this compares to the filter() and slice() operations that we'll
+        // use for the next two data transforms.
+      } 
+      else if (options.sorted === 'desc') {
+        data.sort(function (a, b) { return  b.Victims- a.Victims; });
+        // Note that this is just the opposite of the above.
+      }
+        }
+    // Filter victims by region
+    // Filter: Step1 : Preparing data
+  
+    var data = conflict[0];
+    //putting region in a separate array 
+    // Filter: Step2: Creating the function
+    data = conflict.filter(function (a) { return d.Region > 0; });
```

Comment:
This breaks, because our function is naming its first argument `a`, but is trying to use an argument named `d`. We get to name our arguments whatever we'd like, only key is to use 'em consistently.